src/components/mediaViewer/mediaViewer.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import BrowserOnly from '@docusaurus/BrowserOnly';
+import './mediaViewer.css';
+
+const ReactImageVideoLightbox = require('react-image-video-lightbox').default;
+
+const TYPE_PHOTO = "photo";
+const TYPE_VIDEO = "video";
+
+const getSrc = (obj) => {
+  return typeof obj === 'string' ? obj : obj.default;
+}
+
+export function MediaViewer({ src, type, alt, thumbnail }) {
+  const [open, setOpen] = useState(false);
+
+  return (
+    <BrowserOnly>
+      {() => {
+        const thumbnailSrc = getSrc(thumbnail || src);
+        let contentSrc = getSrc(src);
```

Comment:
Seems like not needed to call `getSrc` twice in case of no `thumbnail`.
Can be updated to:
```javascript
let contentSrc = getSrc(src);
const thumbnailSrc = thumbnail ? getSrc(thumbnail) : contentSrc;
```